File: apps/console/BasketballConsole/BasketballConsole/Models/ValueObjects/GamePlay.cs
using NBA.Repo.Tables;

namespace NBA.Models.ValueObjects
{
    public class GamePlay
    {
        public int Quarter { get; set; }

        public string Type { get; set; }

        public int? Points { get; set; }

        public TimeSpan At { get; set; }

        internal static GamePlay FactorFrom(Play playTable, int quarter)
        {
            return new()
            {
                Quarter = quarter,
                Type = playTable.Type,
                Points = playTable.Points,
                At = playTable.At.Value
            };
        }
    }
}

File: Vue with MongoDB/client/nba-nuxt-app/models/GamePlay.ts
export class GamePlay {
    quarter: number;
    type: string;
    points?: number | null;
    at: string;

    constructor(quarter: number, type: string, points: number | null | undefined, at: string) {
        this.quarter = quarter;
        this.type = type;
        this.points = points ?? null;
        this.at = at;
    }
}
